### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by titaniumbones**
This merges the harvesting-toolkit into this repo and updates README. Please **really review this** before mergine, it does make some real changes.

### CODE REVIEW

**Comment by titaniumbones** [comment]
the conflicts are in research.md, which this PR removes. So, if you want to merge it, please indicate here, I can always just do it from the CLI.

**Comment by b5** [comment]
This PR has a question about generating uuids. In Philly I had people use this handy web tool: https://www.uuidgenerator.net it can generate batches of uuids as well as individual ones.

Otherwise, looks great!

**Comment by titaniumbones** [comment]
@b5 thx added in latest commit.

**Review by b5 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,36 +1,35 @@
...
-- canvas the resources of a given government agency, identifying important URLs.
-- sort them by whether their data can be automatically captured by the Internet Archive webcrawler
-- nominate urls for the Internet Archive if they are crawlable and mark them as "uncrawlable" if they're not
+### 2. Research
+Resarchers inspect the "uncrawlable" list to confirm that seeders' assessments were correct (that is, that the URL/dataset is indeed uncrawlable. [Research.md](research.md) describes this process in more detail. 
 
-**The [Researchers](research.md)**
-- evaluate a url to see if it is indeed crawlable
-- make a plan for how the data might be accessed
+*Often this step is incorporated into either "Seeding and Sorting" or "Harvesting".
```
Comment (dcwalk) [inline README.md]: Think there is a hanging asterisk here

File: README.md [hunk 2]
```diff
@@ -1,36 +1,35 @@
...
 
-**The [Harvesters](harvesting.md)**
-- download uncrawlable data. Often is it easiest for the person who researched an entry to also harvest it, obtaining technical assistance from fellow researchers/harvesters as needed
-- create brief metadata json file and notes for the next steps
+### 3. [Harvesting](harvesting-toolkit)
+Harvesters take the "uncrawlable" data and try to figure out how to capture it. This is a complex task which can require substantial technical expertise, and which requires different techniques for different tasks. Harvesters should see the included [Harvesting Toolkit](./harvesting-toolkit) for more details and tools. **Group Leaders should familiarize themselves with this process before the start of the event.
```
Comment (dcwalk) [inline README.md]: Looks like the bold is also hanging

**Comment by dcwalk** [comment]
Just a some small fixes. Also-- I think removing research is okay. RE: the merge conflict, I think it is because you are behind master in this branch, you could rebase on master before merge, which should resolve (but may cause a lot of other work)

**Comment by titaniumbones** [comment]
ok, tried to rebase (!!) hopefully this is good to go now. Research.md still there, w/ some edits by @murphyofglad  that make it probably a little more useful.

**Comment by b5** [comment]
Looks good to me! Is anything holding back a merge at this point?

**Comment by dcwalk** [comment]
I can't merge, otherwise I would 😁. But seriously-- maybe review form @librlaurie would be helpful as she coordinated this workflow

**Comment by b5** [comment]
Lol ok merging. Someone can yell / rollback later :)

**Pull request merged**